lib/media/keyboard.ts

```diff
@@ -3,10 +3,8 @@ import store, { KeyboardShortcutConfig } from '../main/store'
 import { STORE_KEYS } from '../constants/store-keys'
 import { getNativeBinaryPath } from './native-interface'
 import { BrowserWindow } from 'electron'
-import { audioRecorderService } from './audio'
-import { voiceInputService } from '../main/voiceInputService'
+import { itoSession } from '../main/itoSession'
```

Comment:
keyboard now starts itoSession instead of interactionManager and voiceInputService. Both are managed by itoSession now